dodo.py

```diff
@@ -319,7 +319,7 @@ def task_dynalite():
     dynalite db for testing and local runs: start, stop
     '''
     cmd = f'{DYNALITE} --port {CFG.DYNALITE_PORT}'
-    msg = f'dyanlite server started on {CFG.DYNALITE_PORT} logging to {CFG.DYNALITE_FILE}'
+    msg = f'dynalite server started on {CFG.DYNALITE_PORT} logging to {CFG.DYNALITE_FILE}'
```

Comment:
Minor type correction while in the neighborhood.  Feel free to call it out and I will revert and follow-up with a separate PR.

Reply:
that's fine; it was hard to see the diff